war/src/main/webapp/css/style.css

```diff
@@ -1932,4 +1931,14 @@ table#legend-table td {
     color: #8a6d3b;
     background-color: #fcf8e3;
     border-color: #faebcc;
+}
+
+body.main-panel-only #side-panel {
+    width: 0 !important;
+    padding: 0 !important;
+    overflow: hidden !important;
+}
+body.main-panel-only #main-panel {
+    margin: 0 auto !important;
+    max-width: 75em !important;
```

Comment:
How does it work with? https://github.com/jenkinsci/jenkins/pull/2029

We should merge https://github.com/jenkinsci/jenkins/pull/2029 before this.

/cc @daniel-beck